packages/@cdktf/provider-generator/lib/get/generator/models/attribute-type-model.ts

```diff
@@ -94,8 +98,20 @@ export class AttributeTypeModel {
     return this.name === TokenizableTypes.NUMBER;
   }
 
+  public get isStringSet(): boolean {
+    return this.isSet && this.name === TokenizableTypes.STRING_LIST;
```

Comment:
I think checking `_type` would be more clear

Reply:
I took that from the `isStringList()` check – I suspect that `name()` handles further cases (i.e. single item lists) but they may also not apply for sets.
But I agree that this whole class could be refactored 😅